artemis-docker/docker-run.sh

```diff
@@ -33,7 +33,11 @@ fi
 
 if ! [ -f ./etc/broker.xml ]; then
     CREATE_ARGUMENTS="--user ${ARTEMIS_USER} --password ${ARTEMIS_PASSWORD} --silent ${LOGIN_OPTION} ${EXTRA_ARGS}"
-    echo CREATE_ARGUMENTS=${CREATE_ARGUMENTS}
+    if [[ ${SUPPRESS_CREDENTIALS,,} == "true" ]]; then
+      echo CREATE_ARGUMENTS="--user XXXXX --password XXXXX --silent ${LOGIN_OPTION} ${EXTRA_ARGS}"
```

Comment:
wouldn't be simpler to just not echo CREATE_ARGUMENTS?

just open a JIRA, and remove this line... no need to print that.

Reply:
Yes, it would 👍